projects/plugins/jetpack/modules/stats.php

```diff
@@ -241,8 +241,9 @@ function stats_admin_menu() {
 	}
 
 	// phpcs:ignore WordPress.Security.NonceVerification.Recommended
-	if ( ( new Host() )->is_woa_site() || ! Stats_Options::get_option( 'enable_odyssey_stats' ) || isset( $_GET['noheader'] ) ) {
+	if ( get_option( 'wpcom_admin_interface' ) !== 'wp-admin' && ( ( new Host() )->is_woa_site() || ! Stats_Options::get_option( 'enable_odyssey_stats' ) || isset( $_GET['noheader'] ) ) ) {
```

Comment:
I might be wrong but I feel like the condition should be combined with `( new Host() )->is_woa_site()`, like:

```suggestion
	if ( (  get_option( 'wpcom_admin_interface' ) !== 'wp-admin' && ( new Host() )->is_woa_site() ) || ! Stats_Options::get_option( 'enable_odyssey_stats' ) || isset( $_GET['noheader'] ) ) ) {
```

Reply:
You're right, thanks for noticing it